KBaseMetagenomes.spec

```diff
@@ -3,16 +3,20 @@
 
 module KBaseMetagenomes {
 
-    /*
-    Data type for binned contigs:
-
-    typedef struct {
-        ...
-
-    } BinnedContigs;
+    typedef string bin_id;
+    typedef string contig_id;
 
+    /*
+     Reference to an assembly object
+     @id ws KBaseGenomeAnnotations.Assembly
     */
+    typedef string contig_obj_ref;
 
+    typedef structure {
+        list<bin_id> bin_ids;
+        mapping<bin_id, list<contig_id>> bin2contig_id;
```

Comment:
I'm wondering if we should make some of these objects structures instead of simple lists/maps so we can attach information in the future if needed (for instance, we don't need coverage/gc or other stats now, but we may want some of those in the future).

Maybe something like:

typedef struct {
    bin_id id;
    /* ... this gives us a place to add more fields later */
} ContigBin;

In fact, in the ContigBin struct, you could add that list of contigs that are mapped to that bin...